web/templates/topic_edit_form.hbs

```diff
@@ -1,11 +1,12 @@
 {{! Client-side Handlebars template for rendering the topic edit form. }}
 
 <form class="topic_edit_form">
```

Comment:
There's one thing that looks wrong I noticed while testing this PR: The loading spinner for resolving a topic is too close to the left once you submit.

I'm not sure if that's new, but can you put up a PR tweaking that as a follow-up?

Reply:
Sure thing, I'll tackle this as a follow-up PR.